docs/serverless-v2/08-03-bind-service-instance.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Bind a Service Instance to a lambda
+type: Tutorials
+---
+
+This tutorial shows how you can bind a sample instance of the Redis service to a lambda. After completing all steps, you will get the lambda with encoded Secrets to the service. You can use them for authentication when you connect to the service to implement custom business logic of your lambda.
+
+To create the binding, you will use ServiceBinding and ServiceBindingUsage custom resources (CRs) managed by the Service Catalog.
+
+>**NOTE:** To learn more about binding Service Instances to applications in Kyma, read [this](/components/service-catalog/#details-provisioning-and-binding) document.
+
+## Prerequisites
+
+This tutorial is based on an existing lambda. To create one, follow the [Create a lambda](#tutorials-create-a-lambda) tutorial.
+
+## Steps
+
+Follows these steps:
+
+1. Export these variables:
+
+    ```bash
+    export NAME={LAMBDA_NAME}
+    export NAMESPACE=serverless
+    ```
+
+    > **NOTE:** Lambda takes the name from the Function CR name. The ServiceInstance, ServiceBinding, and ServiceBindingUsage CRs can have different names, but for the purpose of this tutorial, all related resources share a common name defined under the **NAME** variable.
+
+2. Create a ServiceInstance CR. You will provision and use [Redis](https://redis.io/) with its `micro` plan:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.k8s.io/v1beta1
+    kind: ServiceInstance
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      clusterServiceClassExternalName: redis
+      clusterServicePlanExternalName: micro
+      parameters:
+        imagePullPolicy: Always
+    EOF    
+    ```
```

Comment:
Is the Redis available by default in Kyma? Maybe we should say that you need a ServiceInstance first and if not point to the proper tutorial?

Reply:
By default not. Also we haven't any tutorial for provisioning sample serviceClass in our documentation, so here we need to point to external docs, but we have a tutorial to bind in cluster Redis ServiceClass by Addon, but doc for that is outdated :( So my proposition is write as you wrote, user must firstly have Redis ServiceClass and for that we can add small yaml for provisioning Addon with Redis.

Reply:
As discussed with @magicmatatjahu , there will be an additional step on adding yaml for provisioning an Addon with Redis.

Reply:
Ok :smile: